suppressed-cves.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd">
+    <suppress>
+        <notes><![CDATA[file name: dropwizard-views-1.4.0-SNAPSHOT.jar]]></notes>
+        <gav regex="true">^io\.dropwizard:dropwizard-views(-.*)?:.*$</gav>
```

Comment:
I think it's safe to ignore CVE-2015-3378 completely and not just for this artifact.